contracts/axelar-gateway/src/event.rs

```diff
@@ -30,7 +30,19 @@ pub(crate) fn execute_contract_call(env: &Env, message_id: String) {
     env.events().publish(topics, ());
 }
 
-pub(crate) fn rotate_signers(env: &Env, signers: WeightedSigners) {
-    let topics = (symbol_short!("rotated"),);
-    env.events().publish(topics, (signers,));
+pub(crate) fn rotate_signers_set(
+    env: &Env,
+    epoch: u64,
+    signers: WeightedSigners,
+    signer_hash: BytesN<32>,
+) {
+    let topics = (symbol_short!("rotated"), epoch, signer_hash);
```

Comment:
breaking change to the gateway event. Not needed for this PR, so we can avoid changing it

Reply:
updated